content/manuals/compose/how-tos/networking.md

```diff
@@ -59,6 +59,12 @@ Within the `web` container, your connection string to `db` would look like
 `postgres://db:5432`, and from the host machine, the connection string would
 look like `postgres://{DOCKER_IP}:8001` for example `postgres://localhost:8001` if your container is running locally.
 
+## Container to container communication
+
+Those familiar with using `http://localhost:3000` to reference a local development URL, use `http://servicename:3000` instead when using docker compose.
+
+Example: In local development you have a frontend node app running at `http://localhost:3000` which references a backend api running at `http://localhost:8080/api`.  In Docker networking, you would replace the node app api reference to the backend api with `http://apiservice:8080/api`. Similarly, the frontend app would be referenced by service name `http://myapp:3000`
```

Comment:
We should clarify that this DNS resolution only works between compose services. E.g., if you would like to view your node frontend app in your browser, you would still need to publish the port from the container to the host and access it via loopback.

Reply:
This is a great idea, wouldnt want to confuse learners further by having people type the wrong port or something like`http://servicename:3000`.  I think the sentence you wrote should be good, because the content just above in `/compose/how-tos/networking/` describes host ports and container ports.  Should I add a small example if it helps?

We should clarify that this DNS resolution only works between compose services. For example, if you would like to view your node frontend app in your browser running at container port 80 and host port 3000,  you would still need to publish the port from the container to the host then you can view the node app at `http://localhost:3000`

 Also I haven't heard of loopback before, what should I look into and should I mention loopback as well?